### PR TITLE
system/elektra/version|modules should have restrict/write, restrict/remove #2109

### PR DESCRIPTION
**Pull request by Chemin1**
## Basics

Do not describe the purpose here but:

- [x] Short descriptions should be in the release notes (added as entry in
      `doc/news/_preparation_next_release.md` which contains `_(my name)_`)
      **Please always add something to the the release notes.**
- [ ] Longer descriptions should be in documentation or in design decisions.
- [x] Describe details of how you changed the code in commit messages
      (first line should have `module: short statement` syntax)
- [x] References to issues, e.g. `close #X`, should be in the commit messages.

## Checklist

Check relevant points but **please do not remove entries**.
For docu fixes, spell checking, and similar none of these points below
need to be checked.

- [ ] I added unit tests
- [x] I ran run_nokdbtests locally and everything went fine
- [ ] affected documentation is fixed
- [ ] I added code comments, logging, and assertions (see doc/CODING.md)
- [ ] meta data is updated (e.g. README.md of plugins)

## Review

@markus2330 please review my pull request

### CODE REVIEW

**Comment by markus2330** [comment]
Thank you for creating the PR! It seems like the build server finds some problems, please fix them.

File: src/libs/elektra/plugin.c [hunk 1]
```diff
@@ -413,7 +413,8 @@ static int elektraVersionGet (Plugin * handle ELEKTRA_UNUSED, KeySet * returned,
 	KeySet * info = elektraVersionKeySet ();
 	keySetMeta(info->array[0], "restrict/write", "1");
 	keySetMeta(info->array[0], "restrict/remove", "1");
-	for (size_t i = 1; i < info->size; i++) {
+	for (size_t i = 1; i < info->size; i++)
```
Comment (sanssecours) [inline src/libs/elektra/plugin.c]: 👋 Hi Dominic, I would really recommend you install [`clang-format`](https://clang.llvm.org/docs/ClangFormat.html#clangformat) and format the file with something like

```sh
clang-format -i src/libs/elektra/plugin.c
```

, the script [`reformat-source`](https://master.libelektra.org/scripts/reformat-source), or something similar. Formatting the file by hand is really error prone, and will most probably not be a fun experience.

Comment (markus2330) [reply src/libs/elektra/plugin.c]: Thank you for helping! We should have similar text as output in the build server, see  #2336

**Comment by markus2330** [comment]
Thank you for fixing this problem!

File: src/libs/elektra/plugin.c (hunk 1 above)
Comment (Chemin1) [reply src/libs/elektra/plugin.c]: Thank you for the hint! I did not know about this script.

**Pull request merged**